lib/udf_findhotel_utils.rb

```diff
@@ -222,10 +222,14 @@ def get_label_value(label, *keys):
             import hashlib
             import json
 
+            def hotel_city_name_lcase(hcn):
+                if hcn != None:
+                 return(hcn.lower())
+
             key = {
                 "campaign_id": campaign_id,
                 "ad_group_id": ad_group_id,
-                "destination": hotel_city_name.lower(),
+                "destination": hotel_city_name_lcase(hotel_city_name),
```

Comment:
better:

```
"destination": hotel_city_name and hotel_city_name.lower()
```

Reply:
sorry @germangh not fully sure what you mean with this one?
Are you suggesting to change the name of function input from **hcn** to **hotel_city_name**?